pkg/sql/conn_executor.go

```diff
@@ -3687,3 +3759,56 @@ func init() {
 		return planGistFromCtx(ctx)
 	})
 }
+
+type jobsInfo struct {
```

Comment:
I think you can come up with a better name for this structure. `txnJobsCollection`?

Reply:
yes, done.